arangod/Aql/Query.h

```diff
@@ -306,6 +306,8 @@ class Query {
   /// @brief pass-thru a resolver object from the transaction context
   CollectionNameResolver const& resolver();
 
+  ExecutionPlan* stealPlan() { return std::move(preparePlan()); }
```

Comment:
```suggestion
#ifdef ARANGODB_USE_GOOGLE_TESTS
  ExecutionPlan* stealPlan() { return std::move(preparePlan()); }
#endif
```
this is not an option in production code